<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>558</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plan a trip!</string>
    </property>
   </widget>
   <widget class="QListWidget" name="campusTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>351</width>
      <height>331</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>12</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Campuses</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Souvenirs available at selected campus</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>401</width>
      <height>461</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>401</width>
      <height>461</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>row1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>row2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>row3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>row4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>row5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Price</string>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Item</string>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>$25</string>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>A beanie</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>$5,000</string>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>A horse</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>$24</string>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>Pants</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>$12</string>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>A pant</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>priceless</string>
     </property>
     <property name="textAlignment">
      <set>AlignTrailing|AlignVCenter</set>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>:)</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Home: </string>
    </property>
   </widget>
   <widget class="QComboBox" name="homeBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>291</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QToolButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>13</y>
      <width>29</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../saveIcon.png</normaloff>../saveIcon.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>29</width>
      <height>29</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionLog_Out"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLogin">
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionLog_Out">
   <property name="text">
    <string>Log Out</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionAdd_from_file">
   <property name="text">
    <string>Add from file...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
